src/polymer/polymer-element.ts

```diff
@@ -96,6 +104,7 @@ export class ScannedPolymerElement extends ScannedElement {
     if (options.properties) {
       options.properties.forEach((p) => this.addProperty(p));
     }
+    this.listeners = this.listeners || [];
```

Comment:
To make it consistent with the other lists, just initialize it at the declaration (see `Slot[]`)